tests/tests.json

```diff
@@ -278,5 +278,29 @@
 	"rea": "r_inhomo",
 	"params": {},
 	"parallelization": ["mpi"]
+    },
+    "drift-dirichlet-bdf1r": {
+	"app": "drift",
+	"dir": "driftbcs",
+	"usr": "d_inhomo",
+	"rea": "d_inhomo",
+	"params": {"17": -11},
+	"parallelization": ["mpi"]
+    },
+    "drift-neumann-bdf1r": {
+	"app": "drift",
+	"dir": "driftbcs",
+	"usr": "n_inhomo",
+	"rea": "n_inhomo",
+	"params": {"17": -11, "11":500},
```

Comment:
Not a blocking issue, but for the future try to remember to put spaces after the colons (i.e. `"11": 500`).